_maps/RandomRuins/LavaRuins/lavaland_surface_scp_facility.dmm

```diff
@@ -616,6 +616,13 @@
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
 	},
 /area/ruin/unpowered)
+"uS" = (
+/obj/structure/rack,
+/obj/item/gun/ballistic/automatic/pistol/m1911{
+	spawnwithmagazine = 0
```

Comment:
spawnwithmagazine = 1

Reply:
Womp womp why not just let cargo buy or print mags for the m1911's???????


I'll swap the 0 for a 1 after work

Reply:
they have no reason to be able to, since they have no way to get the gun.